src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -74,7 +81,7 @@ class __attribute__((visibility("default"))) TvgLottieAnimation
     }
 
     // Render methods
-    bool load(string data, string mimetype, int width, int height, string rpath = "")
+    bool load(string data, string mimetype, int width, int height, string selector = "", string rpath = "")
```

Comment:
The selector is not compatible with other engines. Could you clarify what values the selector is supposed to receive from the user? Otherwise we can ignore it if it's an optional.

Reply:
The value `selector` has to be HTML element selector.

This is required by `WGPUSurfaceDescriptorFromCanvasHTMLSelector`, to initialize browser based WGPU surface, which means this is not an optional.

In SW Engine, no need to have selector value. We can skip or pass and forget it.

Reply:
@tinyjin 
1. We don't use the selector in the wgpu native systems.
2. What string value is the user supposed to send?

Reply:
1. This is part of WGPU surface creation:

```cpp
WGPUSurfaceDescriptorFromCanvasHTMLSelector canvasDesc{};
canvasDesc.chain.next = nullptr;
canvasDesc.chain.sType = WGPUSType_SurfaceDescriptorFromCanvasHTMLSelector;
canvasDesc.selector = this->selector.c_str(); // This requires selector
WGPUSurfaceDescriptor surfaceDesc{};
surfaceDesc.nextInChain = &canvasDesc.chain;
surface = wgpuInstanceCreateSurface(instance, &surfaceDesc);
```

2. Target Canvas that we want to draw on

```html
<canvas class="thorvg" style="width: 100%; height: 100%;" />
```

With this canvas, users should pass `.thorvg` value, which is class selector.
Please Note, binding users should pass the data, but library users don't have to handle this value, because JavaScript side will cover the logic(canvas creation, passing selector).